### PR TITLE
add report problem button

### PR DESCRIPTION
**Pull request by KongPobW**

### CODE REVIEW

**Comment by chinathaip** [comment]
Review pls sang q @Lxkas @NayHtetKyaw

File: src/components/core/ApplicationHeader.tsx [hunk 1]
```diff
@@ -159,6 +159,13 @@ export default function ApplicationHeader({ opened, setOpened, theme }: Applicat
 								/>
 							</div>
 						</div>
+
+						{/* Report Problem */}
+						<div className="relative cursor-pointer">
```
Comment (Lxkas) [inline src/components/core/ApplicationHeader.tsx]: relative, cursor-pointer is not needed, unless there's a specific reason you added it?

File: src/components/core/ApplicationHeader.tsx [hunk 2]
```diff
@@ -159,6 +159,13 @@ export default function ApplicationHeader({ opened, setOpened, theme }: Applicat
...
+							<Link href="https://forms.office.com/r/z48ExG8dPs">
+								<UnstyledButton className="w-40 h-full p-2">Report Problem</UnstyledButton>
```
Comment (Lxkas) [inline src/components/core/ApplicationHeader.tsx]: Why UnstyledButton? Use the subtle or outline variant of the Button component

File: src/components/core/ApplicationHeader.tsx [hunk 3]
```diff
@@ -159,6 +159,13 @@ export default function ApplicationHeader({ opened, setOpened, theme }: Applicat
 								/>
 							</div>
 						</div>
+
+						{/* Report Problem */}
+						<div className="relative cursor-pointer">
+							<Link href="https://forms.office.com/r/z48ExG8dPs">
```
Comment (Lxkas) [inline src/components/core/ApplicationHeader.tsx]: Probably better to make this open a new tab, pass the target attribute accordingly

**Review by Lxkas — CHANGES_REQUESTED** [review]
pls address all comments and make the necessary changes ty 💖

File: src/components/core/ApplicationHeader.tsx (hunk 2 above)
Comment (chinathaip) [reply src/components/core/ApplicationHeader.tsx]: For references @KongPobW 
https://mantine.dev/core/button/

File: src/components/core/ApplicationHeader.tsx [hunk 4]
```diff
@@ -161,10 +162,10 @@ export default function ApplicationHeader({ opened, setOpened, theme }: Applicat
 						</div>
 
 						{/* Report Problem */}
-						<div>
-							<Link href="https://forms.office.com/r/z48ExG8dPs">
-								<UnstyledButton className="w-40 h-full p-2">Report Problem</UnstyledButton>
-							</Link>
+						<div className="ml-3">
```
Comment (anascence) [inline src/components/core/ApplicationHeader.tsx]: looks cool cool, just one thing I want to ask to (add) na!
I think it's not responsive. or it is just on my device.

Comment (Lxkas) [reply src/components/core/ApplicationHeader.tsx]: What device? Screen size? Browser? Not sure if the margins broke smth

**Pull request closed without merging**